moveit_ros/perception/mesh_filter/src/depth_self_filter_nodelet.cpp

```diff
@@ -115,7 +115,11 @@ void mesh_filter::DepthSelfFiltering::filter(const sensor_msgs::ImageConstPtr& d
 
   // Handling of two possible encodings of a depth image: 16UC1 and 32FC1
   if (depth_msg->encoding == sensor_msgs::image_encodings::TYPE_16UC1)
+  {
+    ROS_WARN_STREAM_THROTTLE(1.0, "The input depth image is encoded in the deprecated 16UC1 format, please consider "
+                                  "converting it to 32FC1 according to ROS REP-118!");
```

Comment:
I still think the `16UC1` format is fully covered by REP-118 and should be supported as it doesn't cost us any effort. If this warning is issued for that format at all, it should be issued only once, i.e. using `ROS_WARN_STREAM_ONCE`.